addons/web/static/src/core/emoji_picker/emoji_picker.xml

```diff
@@ -23,8 +23,8 @@
                     <span class="o-EmojiPicker-empty">😢</span>
                     <span class="fs-5 text-muted">No emoji match your search</span>
                 </t>
-                <t t-if="recentEmojis.length > 0 and !searchTerm">
-                    <t t-call="web.EmojiPicker.section">
+                <t t-if="recentEmojis.length > 0">
+                    <t t-call="web.EmojiPicker.section" t-if="!searchTerm">
```

Comment:
(Nitpicking) : it can be done like this to increase readability :
`<t t-if="!searchTerm" t-call="web.EmojiPicker.section">`